Assets/MirageXR/Managers/ExceptionManager/ExceptionManager.cs

```diff
@@ -0,0 +1,59 @@
+using i5.Toolkit.Core.VerboseLogging;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace MirageXR
+{
+
+   /// <summary>
+   /// Class <c>ExceptionManager</c> registers for all serious exceptions and opens a user dialogue 
+   /// to ask to send a report to sentry.io; the main exceptions registered for are:
+   /// Null Reference, Divide by Zero, Out of Memory, Index Out of Range.
+   /// </summary>
+
+   public class ExceptionManager : MonoBehaviour
+   {
+
+       // the Sentry instance
+       public SentrySdk sentry; 
+
+       void Awake()
+       {
+           AppLog.Log("Installing hook for exceptions", LogLevel.INFO );
+           sentry ??= new GameObject("ExceptionManagerSentry").AddComponent<SentrySdk>();
+           sentry.Dsn = "https://b23911205078e7a81bf1489e8aa0fabe@o4506320008118272.ingest.sentry.io/4506320009428992"; // our DSN
```

Comment:
move to `const`

Reply:
can do the next couple of days, but would propose we merge this in now, so we get improved tracking with version 2.5?

Reply:
@cawke agree?